packages/crypto/src/ed25519.rs

```diff
@@ -288,7 +288,8 @@ mod tests {
             // ed25519_verify() works
             assert!(
                 ed25519_verify(&message, &signature, &public_key).unwrap(),
-                format!("verify() failed (test case {})", i)
+                "verify() failed (test case {})",
+                i
```

Comment:
Woah, didn't knew this was possible. Thanks.

Reply:
There is a new clippy rule that warns if you put anything else in the message argument than a format expression (e.g. Strings).